openstack/db/v1/configurations/results.go

```diff
@@ -0,0 +1,142 @@
+package configurations
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Config represents a configuration group API resource.
+type Config struct {
+	Created              string
```

Comment:
Can we set these as `time.Time` and convert them in `Extract` similar to here: https://github.com/rackspace/gophercloud/blob/d783aa72ca0531c6d07f76c84fc73083a5e8af7f/rackspace/rackconnect/v3/cloudnetworks/results.go#L61